frontend/constants/serviceTemplates.ts

```diff
@@ -7,394 +7,323 @@ import { StakingProgramId } from '@/enums/StakingProgram';
-  //         use_staking: false,
-  //         use_mech_marketplace: false,
-  //         cost_of_bond: 1,
-  //         monthly_gas_estimate: 1000,
-  //         fund_requirements: {
-  //           agent: 1000,
-  //           safe: 1000,
-  //         },
-  //       },
-  //       [CHAINS.BASE.middlewareChain]: {
-  //         staking_program_id: StakingProgramId.OptimusAlpha, // default, may be overwritten
-  //         nft: 'bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq',
-  //         // rpc: 'http://localhost:8545',
-  //         agent_id: 40,
-  //         threshold: 1,
-  //         use_staking: false,
-  //         use_mech_marketplace: false,
-  //         cost_of_bond: 1,
-  //         monthly_gas_estimate: 1000,
-  //         fund_requirements: {
-  //           agent: 1000,
-  //           safe: 1000,
-  //         },
-  //       },
-  //     },
-  //   },
-  {
-    agentType: AgentType.Memeooorr,
-    name: 'Memeooorr',
-    hash: 'bafybeicy44nktxwngev2dlgathl2ir644btp76q63wpcdteim66t6lyjuu',
-    description: 'Memeooorr @twitter_handle', // should be overwritten with twitter username
-    image:
-      'https://gateway.autonolas.tech/ipfs/QmQYDGMg8m91QQkTWSSmANs5tZwKrmvUCawXZfXVVWQPcu',
-    service_version: 'v0.2.0-alpha16',
-    home_chain: MiddlewareChain.BASE,
-    configurations: {
-      [MiddlewareChain.BASE]: {
-        staking_program_id: StakingProgramId.MemeBaseAlpha2, // default, may be overwritten
-        nft: 'bafybeiaakdeconw7j5z76fgghfdjmsr6tzejotxcwnvmp3nroaw3glgyve',
-        rpc: 'http://localhost:8545', // overwritten
-        agent_id: 43,
-        threshold: 1,
-        use_staking: true,
-        cost_of_bond: +parseEther(50),
-        monthly_gas_estimate: +parseEther(0.03),
-        fund_requirements: {
-          // zero address means native currency
-          [ethers.constants.AddressZero]: {
-            agent: +parseEther(0.001),
-            safe: +parseEther(0.0125),
-          },
+} as const;
+
+export const MEMEOOORR_BASE_TEMPLATE: ServiceTemplate = {
+  agentType: AgentType.Memeooorr,
+  name: 'Memeooorr',
+  hash: 'bafybeihgaoi7u4ryeopz3ujzeplmheqnsop7mh25nvkvzfp747uaurru6a',
+  description: 'Memeooorr @twitter_handle', // should be overwritten with twitter username
+  image:
+    'https://gateway.autonolas.tech/ipfs/QmQYDGMg8m91QQkTWSSmANs5tZwKrmvUCawXZfXVVWQPcu',
+  service_version: 'v0.2.0-alpha16',
+  home_chain: MiddlewareChain.BASE,
+  configurations: {
+    [MiddlewareChain.BASE]: {
+      staking_program_id: StakingProgramId.MemeBaseAlpha2, // default, may be overwritten
+      nft: 'bafybeiaakdeconw7j5z76fgghfdjmsr6tzejotxcwnvmp3nroaw3glgyve',
+      rpc: 'http://localhost:8545', // overwritten
+      agent_id: 43,
+      threshold: 1,
+      use_staking: true,
+      cost_of_bond: +parseEther(50),
+      monthly_gas_estimate: +parseEther(0.045),
+      fund_requirements: {
+        [ethers.constants.AddressZero]: {
+          agent: +parseEther(0.00625),
```

Comment:
it was updated to `0.001` in fix/meme-staging, maybe worth pulling
as well as monthly_gas_estimate